src/sys/signal.rs

```diff
@@ -582,6 +582,34 @@ impl SigSet {
         })
     }
 
+    /// Wait for a signal
+    ///
+    /// # Return value
+    /// If `sigsuspend(2)` is interrupted (EINTR), this function returns `Ok`.
+    /// If `sigsuspend(2)` set other error, this function returns `Err`.
+    ///
+    /// For more information see the
+    /// [`sigsuspend(2)`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/sigsuspend.html).
+    #[cfg(any(
+        target_os = "linux",
+        target_os = "android",
+        target_os = "hurd",
+        target_os = "nto",
```

Comment:
Please remove this as [we are not sure if we should support this target](https://github.com/nix-rust/nix/pull/2055#discussion_r1285267681)

Reply:
I removed `nto`.